### ISSUE TITLE
[cinema.experience] Countdown to be able to select a folder instead of just a file.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [drrlramsey/xbmc-addons#77]
```
http://forum.xbmc.org/showpost.php?p=677766&postcount=400

SlaveUnit asks - Any way to have the COUNTDOWN option let you select a folder 
and not just 1 file? I have a few I wouldn't mind cycling through. 


```

Original issue reported on code.google.com by `giftie61@gmail.com` on 31 Dec 2010 at 4:10

**Comment by GoogleCodeExporter** [comment]
```
Should be possible... (duplicate what is being used for other videos.)
```

Original comment by `giftie61@gmail.com` on 31 Dec 2010 at 4:11

**Comment by GoogleCodeExporter** [comment]
```
fixed in 1.0.11
```

Original comment by `giftie61@gmail.com` on 4 Mar 2011 at 2:19
- Changed state: **Fixed**
